src/msgpack/MsgPackSettings.cs

```diff
@@ -0,0 +1,118 @@
+using System;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Reflection;
+
+namespace TarantoolDnx.MsgPack
+{
+    [DebuggerStepThrough]
+    public class MsgPackSettings
+    {
+        private static readonly IReadOnlyDictionary<Type, IMsgPackConverter> DefaultConverters = new Dictionary<Type, IMsgPackConverter>
+        {
+            { typeof(bool), new BoolConverter() },
+            { typeof(string), new StringConverter() },
+            { typeof(byte[]), new BinaryConverter() },
+
+            { typeof(float), new FloatConverter() },
+            { typeof(double), new FloatConverter() },
+
+            { typeof(byte), new IntConverter() },
+            { typeof(sbyte), new IntConverter() },
+            { typeof(short), new IntConverter() },
+            { typeof(ushort), new IntConverter() },
+            { typeof(int), new IntConverter() },
+            { typeof(uint), new IntConverter() },
+            { typeof(long), new IntConverter() },
+            { typeof(ulong), new IntConverter() },
+        };
+
+        private static readonly ConcurrentDictionary<Type, IMsgPackConverter> GeneratedConverters = new ConcurrentDictionary<Type, IMsgPackConverter>();
+
+        private static readonly IMsgPackConverter<object> nullConverter = new NullConverter();
```

Comment:
We should make a convention about private fields. Should we start all of them from _?